src/resources/basis.js

```diff
@@ -292,6 +300,7 @@ function basisInitialize(config) {
 
         config.rgbPriority = config.rgbPriority || defaultRgbPriority;
         config.rgbaPriority = config.rgbaPriority || defaultRgbaPriority;
+        config.maxRetries = config.maxRetries || defaultMaxRetries;
```

Comment:
```suggestion
        config.maxRetries = config.hasOwnProperty('maxRetries') ? config.maxRetries : defaultMaxRetries;
```

Since 0 is a valid maxRetries.